mllib/src/main/scala/org/apache/spark/mllib/evaluation/binary/BinaryClassificationMetricComputers.scala

```diff
@@ -24,6 +24,19 @@ private[evaluation] trait BinaryClassificationMetricComputer extends Serializabl
   def apply(c: BinaryConfusionMatrix): Double
 }
 
+/** Accuracy. Defined as 1.0 where there are zero no positive and negative examples. */
+private[evaluation] object Accuracy extends BinaryClassificationMetricComputer {
+  override  def apply(c: BinaryConfusionMatrix): Double = {
```

Comment:
Nit: extra space after override. You could also pull totalTrue into the else block.